apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/RetentionPolicyTest.cs

```diff
@@ -110,6 +110,64 @@ public void LockRetentionPolicy()
             Assert.True(bucket.RetentionPolicy.IsLocked);
         }
 
+        /// <summary>
+        /// Tests for object retention policy.
+        /// </summary>
+        [Fact]
+        public void ObjectRetentionTests()
+        {
+            var client = _fixture.Client;
+            var bucketName = _fixture.GenerateBucketName();
+
+            // Create a bucket with object retention enabled.
+            var createBucketoptions = new CreateBucketOptions { EnableObjectRetention = true };
```

Comment:
So what happens if the code below is tried *without* object retention enabled? I think that would probably be worth another test.